pkg/apis/sessions/session_state.go

```diff
@@ -66,6 +70,7 @@ func (s *SessionState) EncodeSessionState(c encryption.Cipher) (string, error) {
 		// Store only Email and User when cipher is unavailable
 		ss.Email = s.Email
 		ss.User = s.User
+		ss.Groups = s.Groups
```

Comment:
We should get some Groups tests added to the session_state_test.go test table.  Since most users won't have groups, we can leave existing ones that don't have it.

Likely just need to add a couple scenarios with 1 and 2+ groups in a session.

Reply:
Added some test coverage here.